code/datums/diseases/_disease.dm

```diff
@@ -156,7 +156,7 @@ var/list/diseases = typesof(/datum/disease) - /datum/disease
 		if(disease_flags & CAN_RESIST)
 			if(!(type in affected_mob.resistances))
```

Comment:
Nope, it seems type is a read only variable which returns the type of object, handy enough